src/store/modules/keycodes/index.js

```diff
@@ -76,22 +121,22 @@ const mutations = {
   },
   enableSteno(state) {
     state.steno = true;
-    state.keycodes = generateKeycodes(isISO(), state.steno);
+    state.keycodes = generateKeycodes(getOSKeyboardLayout(), state.steno);
   },
   disableSteno(state) {
     state.steno = false;
-    state.keycodes = generateKeycodes(isISO(), state.steno);
-  },
-  enableIso(state) {
-    state.keycodes = generateKeycodes(isISO(), state.steno);
-    if (state.active === 'ANSI') {
-      state.active = 'ISO/JIS';
-    }
+    state.keycodes = generateKeycodes(getOSKeyboardLayout(), state.steno);
   },
-  disableIso(state) {
-    state.keycodes = generateKeycodes(isISO(), state.steno);
-    if (state.active === 'ISO/JIS') {
-      state.active = 'ANSI';
+  updateKeycodeNames(state) {
+    state.keycodes = generateKeycodes(getOSKeyboardLayout(), state.steno);
+    if (isANSI()) {
```

Comment:
Minor issue, but this state change feels like it should probably be in it's own function. You're changing the state of the keycodes and the active tab.

Reply:
Solved in eaee8fa190c69f6ae9c01cc653b27b3de288211d